tests/client.test.js

```diff
@@ -0,0 +1,106 @@
+var common = require('./common.js'),
+    assert = require('assert'),
+    Client = require('../client/client.js'),
+    Persistence = require('../core').Persistence,
+    client, subscriptions;
+
+describe('Client', function() {
+  beforeEach(function(done) {
+    common.startPersistence(done); // clean up
+    client = new Client('joe', Math.random(), 'test', 1);
+    subscriptions = {};
+    presences = {};
+  });
+
+  describe('.storeData and .loadData', function() {
+    describe('subscriptions', function() {
+      it('should store subscribe operations', function(done) {
+        var to = 'presence:/test/account/ticket/1',
+            message = { to: to, op: 'subscribe' };
+        
+        subscriptions[to] = message;
+
+        client.storeData(message);
+
+        client.readData(function(state) {
+          assert.deepEqual(state, {
+            subscriptions: subscriptions,
+            presences: {}
+          });
+          done();
+        });
+      });
+
+      it('should store sync as subscribes', function(done) {
+        var to = 'presence:/test/account/ticket/1',
+            message = { to: to, op: 'sync' };
+        
+        subscriptions[to] = message;
+
+        client.storeData(message);
+
+        client.readData(function(state) {
+          assert.deepEqual(state, {
+            subscriptions: subscriptions,
+            presences: {}
+          });
+          done();
+        });
+      });
```

Comment:
you should add a test here for 'sync' after 'subscribe' and 'subscribe' after 'sync'